documentation/routing/tutorial-routing-annotation.asciidoc

```diff
@@ -30,8 +38,12 @@ public class SomePathComponent extends Div {
 }
 ----
 
-So whenever the user navigates to `http://yourdomain.com/some/path` (assuming the app is running from the root context), either by clicking on links inside the application or by typing the address directly on the address bar, the `SomePathComponent` component would be shown at the page.
+* Assuming your app is running from the root context, when the user navigates to `http://yourdomain.com/some/path`, either by clicking a link in the application or entering the address in the address bar, the `SomePathComponent` component is shown on the page. 
+
+[NOTE]
+If you omit the `@Route` annotation parameter, the route target is derived from the class name. For example, MyEditor becomes "myeditor", PersonView becomes "person" and MainView becomes "".
+
+// I think this NOTE needs further explanation. It is not obvious to me why and how the route target is derived.
```

Comment:
Move this line to new issue in GitHub. We can then ask help form the development team to improve these parts.

Reply:
https://github.com/vaadin/flow-and-components-documentation/issues/404